resources/privacy-and-gdpr.md

```diff
@@ -6,6 +6,251 @@ Before implementing these controls, read the [Adobe Experience Platform Privacy
+
+* `y`
+* `n`
+
+To understand the expected behavior, see the _Update and get collect consent preferences_ table above.
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+You can set the collect consent to one of the following values:
+
+#### Swift
+
+* `y`
+* `n`
+
+#### Objective-C
+
+You can set the collect consent to one of the following values:
+
+* `y`
+* `n`
+
+To understand the expected behavior, see the _Update and get collect consent preferences_ table above.
+{% endtab %}
+{% endtabs %}
+
+### getConsents <a id="getconsents"></a>
+
+You can programmatically view the current collect consent preferences status in a dictionary representation by using the following API.
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+**Syntax**
+
+```java
+public static void getConsents(final AdobeCallback<Map<String, Object>> callback);
+```
+
+* _callback_ - callback invoked with the current consents of the extension. If an `AdobeCallbackWithError` is provided, an `AdobeError`, can be retruned in the eventuality of any error that occured while getting the user consents. The callback may be invoked on a different thread.
+
+**Example**
+
+```java
+Consent.getConsents(new AdobeCallback<Map<String, Object>>() {
+    @Override
+    public void call(Map<String, Object> currentConsents) {
+        if (currentConsents == null) { return; }
+    final Map<String, Object> consents = currentConsets.get("consents");
+    final Map<String, Object> collectConsent = consents.get("collect");
+    final String collectConsentStatus = collectConsent.get("val");
+    // inspect collectConsentStatus
+    }
+});
+```
+{% endtab %}
+
+{% tab title="iOS (AEP 3.x)" %}
+#### Swift
+
+**Syntax**
+
+```swift
+static func getConsents(completion: @escaping ([String: Any]?, Error?) -> Void)
+```
+
+**Example**
+
+```swift
+Consent.getConsents { currentConsents, error in
+	guard error == nil else { return }
+	guard let consents = currentConsents["consents"] as? [String: Any] else { return }
+	guard let collectConsent = consents["collect"] as? [String: Any] else { return }
+	let collectConsentStatus = collectConsent["val"] as? String
+	// inspect collectConsentStatus
+}
+```
+
+#### Objective-C
+
+**Syntax**
+
+```objective-c
+static func getConsents(completion: @escaping ([String: Any]?, Error?) -> Void)
+```
+
+**Example**
+
+```objective-c
+[AEPMobileEdgeConsent getConsents:^(NSDictionary *currentConsents, NSError *error) {
+	if (error) { return; }
+	NSDictionary *consents = currentConsents[@"consents"];
+	NSDictionary *collectConsent = currentConsents[@"collect"];
+	NSString *collectConsentStatus = collectConsent[@"val"];
+	// inspect collectConsentStatus
+}];
+```
+{% endtab %}
+{% endtabs %}
+
+### updateConsents <a id="updateconsent"></a>
+
+Use this example to programmatically update the consent collect for the application user.
+
+> **Note:** After a user has selected collect consent no \(n\), the SDK will not allow you to set the users collect consent to yes \(y\).
```

Comment:
Same comment as above wrt no need for \\: "... selected collect consent `not` (n), the SDK will not allow you to set the user's collect consent to `yes` (y)."